### DISCUSSION TITLE
pre-populated properties

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6824] [General]
Hi,

I would configure an integer property on xyzRow.cs so that when user open form to insert new xyz, it wold be precompiled with:

<max value on DB> + 10

![Schermata 2023-07-05 alle 12 07 46](https://github.com/serenity-is/Serenity/assets/14870650/1f09c3f8-7a57-4e3d-a9f4-6a0be5873035)

**Comment by luongnv6** [comment]
pls review this demo:
https://demo.serenity.is/BasicSamples/SerialAutoNumber#
https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Modules/Dialogs/SerialAutoNumber/SerialAutoNumberPage.ts
